apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -152,6 +152,12 @@ type PersistentHomeConfig struct {
 	// Must be used with the 'per-user'/'common' or 'per-workspace' storage class in order to take effect.
 	// Disabled by default.
 	Enabled *bool `json:"enabled,omitempty"`
+	// Determines whether the init container that initializes the persistent home directory should be disabled.
+	// When the `/home/user` directory is persisted, the init container is used to initialize the directory before
+	// the workspace starts. If set to true, the init container will not be used.
+	DisableInitContainer *bool `json:"initContainerEnabled,omitempty"`
+	// Image is the container image to use for the init container that initializes the persistent home directory.
+	InitContainer *dw.Container `json:"initContainer,omitempty"`
```

Comment:
Is this field supposed to be about specifying the image to use for the init container, or the devworkspace container component? It seems to be the latter in the way it's used. 

Also, small typo in the docs `// Image is the container image to use...`

Reply:
It's the latter, fixed